lib/node_modules/@stdlib/blas/ext/base/snansumpw/src/main.c

```diff
@@ -56,16 +77,12 @@ float stdlib_strided_snansumpw( const int64_t N, const float *X, const int64_t s
 	if ( N <= 0 ) {
 		return 0.0f;
 	}
-	if ( N == 1 || stride == 0 ) {
+	ix = offsetX;
+	if ( strideX == 0 ) {
 		if ( stdlib_base_is_nanf( X[ 0 ] ) ) {
 			return 0.0f;
 		}
```

Comment:
```suggestion
	if ( strideX == 0 ) {
		if ( stdlib_base_is_nanf( X[ ix ] ) ) {
			return 0.0f;
		}
```

Reply:
Consider this case:
```
stdlib_strided_snansumpw_ndarray(  4, [ 1.0f, 0.0f/0.0f, 2.0f, 3.0f ], 0, 1 );
```
For this case, the current implementation will pass the `nanCheck` and return the value of ` N * X[ ix ]` which in this case would be `4 * X[ 1 ]` => `4 * (0.0f/0.0f)`